mm/Kconfig.debug

```diff
@@ -140,8 +140,12 @@ config PTDUMP_DEBUGFS
 	  If in doubt, say N.
 
 config KFENCE
-	bool "KFence"
-	#depends on DEBUG_KERNEL
+	bool "KFence: low-overhead use-after-free and buffer-overflow detector"
+	depends on !KASAN
```

Comment:
A comment about why we disable KASAN would be nice.

Reply:
Will add.